Dockerfile.dev

```diff
@@ -1,7 +1,3 @@
 FROM ghcr.io/runatlantis/atlantis:latest
 COPY atlantis /usr/local/bin/atlantis
-# TODO: remove this once we get this in the base image
-# renovate: datasource=github-releases depName=open-policy-agent/conftest
-ENV DEFAULT_CONFTEST_VERSION=0.38.0
```

Comment:
Is this no longer needed?

Reply:
From what I can tell, it is not needed since it's already in the main image.